### PR TITLE
Add missing test case for Usage

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Ensure that the flutter features are correctly propagated through analytics.

### CODE REVIEW

File: packages/flutter_tools/test/general.shard/reporting/usage_test.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (zanderso) [inline packages/flutter_tools/test/general.shard/reporting/usage_test.dart]: Maybe feature_usage_test.dart, or will other usage tests go here too?

Comment (jonahwilliams) [reply packages/flutter_tools/test/general.shard/reporting/usage_test.dart]: Hopefully!

**Review by zanderso — APPROVED** [review]

File: packages/flutter_tools/test/general.shard/reporting/usage_test.dart (hunk 1 above)
Comment (xster) [reply packages/flutter_tools/test/general.shard/reporting/usage_test.dart]: there's a general.shard/analytics_test.dart already. Can we merge them somehow?

Comment (jonahwilliams) [reply packages/flutter_tools/test/general.shard/reporting/usage_test.dart]: d'oh! updated

**Comment by xster** [comment]
LGTM thanks

**Pull request merged**